src/content/1.7/modules/concepts/templating/vuejs/faq.md

```diff
@@ -0,0 +1,52 @@
+---
+title: FAQ about Vue.js
+menuTitle: FAQ
+weight: 2
+---
+
+### How to send data from PHP to Vue.js?
+
+Use the method `Media::addJsDef([])` to make your data available in the window context.
+If other modules can be present on the same, prefix your names with the module name to avoid name collision.
+
+This is a convenient way to send the links Vue.js could call, generated by the router.
+
+Example:
+
+```php
+    Media::addJsDef([
+        'myVueJsModuleRouteToSomeAction' => $this->context->link->getAdminLink(
+            'AdminAjaxMyVueJsModule',
+            true,
+            [],
+            [
+                'action' => 'RouteToSomeAction',
+                'ajax' => 1,
+            ]
+        ),
+    ]);
+```
+
+### How to run several Vue.js applications on the same page
+
+The only thing to consider is the div ID you want to hook your application on.
+
+By default, Vue.js creates your application with a `<div id="app"></div>`. To avoid it colliding with other applications or modules, replace the references to this ID in the template and the JS development files with something more specific.
```

Comment:
```suggestion
By default, Vue.js creates your application with a `<div id="app"></div>`. To avoid it colliding with other applications or modules, replace references to this ID in the template and JS development files with something more specific.
```

Reply:
Here you go